src/features/marketplace/components/TradeableList.tsx

```diff
@@ -95,7 +98,8 @@ export const TradeableListItem: React.FC<TradeableListItemProps> = ({
 
   const dailyListings = getDailyListings();
 
-  const hasAccess = hasTradeReputation || dailyListings < 1;
+  const hasAccess =
+    state.island.type !== "basic" && (hasTradeReputation || dailyListings < 1);
```

Comment:
This has a bit of a side effect listing of collectibles while on basic Island.

I am a cropkeeper, however the UI shows as I have not reached the required access level.

![listttt](https://github.com/user-attachments/assets/537b6987-48a7-4367-817a-6d649706cfe3)